backoffice/backoffice/users/fixtures/tokens.json

```diff
@@ -0,0 +1,10 @@
+[
```

Comment:
these fixtures are great additions, that greatly improve quality of live when having to create everything from scratch, thanks!